src/services/pseudonym/hooks/index.js

```diff
@@ -40,17 +40,23 @@ const createMissingPseudonyms = (hook) => {
 
 // restricts the return pseudonyms to the users the current user is allowed to retrieve
 const filterValidUsers = (context) => {
-	let validUserIds = [context.params.account.userId];
+	const currentUserId = context.params.account.userId;
+	let validUserIds = [currentUserId];
```

Comment:
this is something like related user id's through course relations, it will be valid later.